pkg/common/util/conn_test.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestServer(t *testing.T) {
+	us := NewUnixSocket("/tmp/us.socket")
+	us.SetContextHandler(func(context string) string {
+		fmt.Println(context)
+		now := "response from server: " + time.Now().String()
+		return now
+	})
+	go us.StartServer()
+	time.Sleep(time.Second * 10)
+}
+
+func TestClient(t *testing.T) {
+	//声明unixsocket
```

Comment:
Remove this comment as it's not meaningful.

Reply:
Updated